packages/internals/src/panic.ts

```diff
@@ -36,6 +36,7 @@ export enum ErrorArea {
   PHOTON_STUDIO = 'PHOTON_STUDIO',
   // Unused since 4.9.0 and now using `LIFT_CLI`
   INTROSPECTION_CLI = 'INTROSPECTION_CLI',
+  // Note: prisma-fmt-wasm was renamed to prisma-schema-wasm in 5.0.0
```

Comment:
What does this refer to here?

Reply:
The error area is something used in our internal error reporting dashboard to categorise Rust panics' "topic areas". It mirrors this `ErrorArea` enum [here](https://github.com/prisma/error-handling-backend/blob/482bf538ec055f4ef41926736d72c5f08e3b9a07/src/types/enums/index.ts#L11-L27).